comp/otelcol/collector-contrib/def/component.go

```diff
@@ -0,0 +1,10 @@
+package collectorContrib
+
+import (
+	"go.opentelemetry.io/collector/otelcol"
```

Comment:
can we make it a module ?

Reply:
Sure thing, pushed a commit to make this available as a go module